numpy/core/code_generators/numpy_api.py

```diff
@@ -354,6 +354,9 @@
     'PyDataMem_SetHandler':                 (304,),
     'PyDataMem_GetHandler':                 (305,),
     # End 1.21 API
+
+    # HPy API:
+    'HPyArray_Zeros':                       (307,),
```

Comment:
I think we shouldn't add HPy functions to the API, but rather, eventually, have a separate function table for the HPy API.

Reply:
Yes, agreed. I want to play with this a bit to move forward quicker: we'd like to prototype integration with Matplotlib port too. However, new table will be the way to go longer term.